pyfr/integrators/dual/pseudo/pseudosteppers.py

```diff
@@ -337,3 +337,316 @@ class DualRK45PseudoStepper(DualRKVdH2RPseudoStepper):
     @property
     def _pseudo_stepper_order(self):
         return 4
+
+
+class DualOptRKPseudoStepper(BaseDualPseudoStepper):
+    @property
+    def _pseudo_stepper_has_lerrest(self):
+        return False
+
+    @property
+    def _stepper_nfevals(self):
+        return 7*self.nsteps
```

Comment:
Lets do a generic dense explicit RK scheme class and then have the magic schemes be a subclass thereof.